ls1intum/Artemis
# `Development`: Fix an issue with gradle in gitignores of programming exercises

### Checklist
#### General



### Motivation and Context


#6404 refactored the gitignores and missed that gradle needs one .jar to function properly. This .jar was previously excluded in the gigignores, but was left out in the mentioned Pr.

### Description

Readds the exception for the cradle-wrapper.jar, so Gradle programming exercises work again.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 Students
- 1 Programming Exercise

1. Log in to Artemis
2. Create a programming exercise using Gradle
3. Check that it is correctly building

### Review Progress






#### Code Review
#### Manual Tests
